apps/sample-imdb/.env.template

```diff
@@ -0,0 +1,4 @@
+XATA_API_KEY=XATA_API_KEY
+XATA_BRANCH=main
+
+OMDB_API_KEY=OMDB API KEY
```

Comment:
Should we maybe add a "how to" section is the readme for how to retrieve this api key?

Reply:
I will as soon as it's ready to be used. I'm not sure if we're keeping the OMDB fallback, tbh... we may not need it anymore